components/message_queue/src/kafka/kafka_impl.rs

```diff
@@ -249,22 +253,62 @@ impl MessageQueue for KafkaImpl {
             })?)
     }
 
-    async fn consume_all(&self, topic_name: &str) -> Result<KafkaConsumeIterator> {
-        info!("Need to consume all data in kafka topic:{}", topic_name);
+    async fn get_earliest_offset(&self, topic_name: &str) -> Result<Offset> {
```

Comment:
How about naming it `fetch_earliest_offset`? `Get` seems a lightweight operation.